upgrade/UPGRADE-v9.1.3-dev.md

```diff
@@ -4,3 +4,9 @@ This guide contains instructions to upgrade from version v9.1.2 to v9.1.3-dev.
 
 **Before you start, don't forget to take a look at [general instructions](https://github.com/shopsys/shopsys/blob/7.3/UPGRADE.md) about upgrading.**
 There you can find links to upgrade notes for other versions too.
+
+- update your composer dependencies (see #project-base-diff)
+  - **\[BC break\]** upgrade `codeception/codeception` to `^4.1.22` so you get rid of [the security problem](https://github.com/advisories/GHSA-4574-qv3w-fcmg)
+    - you might need to update your `Tests\App\Test\Codeception\AcceptanceTester` to respect changes (added strict typehints) in `Tests\FrameworkBundle\Test\Codeception\ActorInterface`
+  - allow plugins in your `composer.json`
```

Comment:
I would like to mention the necessity of the new composer version to emphasize the need for a re-build of possible pre-built optimized docker image

Reply:
something like this? :slightly_smiling_face:  c659c5efc